src/clzones.cpp

```diff
@@ -88,6 +88,12 @@ zone_manager::zone_manager()
     types.emplace( zone_type_id( "CAMP_FOOD" ),
                    zone_type( translate_marker( "Basecamp: Food" ),
                               translate_marker( "Items in this zone will be added to a basecamp's food supply in the Distribute Food mission." ) ) );
+    types.emplace( zone_type_id( "AUTO_EAT" ),
+                   zone_type( translate_marker( "Auto Eat" ),
+                              translate_marker( "Items in this zone will be automatically eaten during a long activity if you get hungry." ) ) );
+    types.emplace( zone_type_id( "AUTO_DRINK" ),
+                   zone_type( translate_marker( "Auto Drink" ),
+                              translate_marker( "Items in this zone will be automatically drunk during a long activity if you get thirsty." ) ) );
```

Comment:
'drunk' has connotations of inebriation, might not be the best word

drank? consumed? quaffed? slurped? idk

Reply:
imbibed?

Reply:
went with consumed